### PR TITLE
Final two SS cards

### PR DESCRIPTION
**Pull request by SuperFlamingTominoHead**
Sorry if I did this wrong, I'm not very good with github yet...

https://i.imgur.com/LC7uHCn.jpg
https://imgur.com/a/hFxsv

### CODE REVIEW

File: pack/ss.json [hunk 1]
```diff
@@ -125,6 +125,39 @@
...
+        "type_code": "resource",
+        "uniqueness": false
+    },
+        {
+        "code": "21009",
+        "deck_limit": 1,
+        "faction_code": "haas-bioroid",
+        "illustrator": "Johan Törnlund",
+        "keywords": "Division",
+        "pack_code": "ss",
```
Comment (SuperFlamingTominoHead) [inline pack/ss.json]: "influence_limit": 15,
        "minimum_deck_size": 40,

File: pack/ss.json [hunk 2]
```diff
@@ -125,6 +125,41 @@
         "title": "Upya",
         "type_code": "program",
         "uniqueness": false
+    },
+           {
```
Comment (gereons) [inline pack/ss.json]: Please align this to the line above

File: pack/ss.json [hunk 3]
```diff
@@ -125,6 +125,41 @@
...
+        "code": "21008",
+        "cost": 5,
+        "deck_limit": 3,
+        "faction_code": "apex",
+        "faction_cost": 5,
+        "flavor": "\"It's only function seems to be repurposing code.\"\n\"Can you kill it?\" she asked.\n\"I can try,\" he said. Guiding his program dead center of the mass, he plunged, piercing the shell before an eruption sent his screen into static. He jacked out, his eyes wide. \"I... I think it absorbed my breaker. And everything else.\"",
```
Comment (gereons) [inline pack/ss.json]: First word should be "Its", not "It's"

File: pack/ss.json [hunk 4]
```diff
@@ -125,6 +125,41 @@
...
+        "illustrator": "Alexandr Elichev",
+        "keywords": "Virtual",
+        "pack_code": "ss",
+        "position": 08,
```
Comment (gereons) [inline pack/ss.json]: Please remove the leading 0

File: pack/ss.json [hunk 5]
```diff
@@ -125,6 +125,41 @@
...
+        "quantity": 3,
+        "side_code": "runner",
+        "text": "[click],[click]: Turn one of your facedown installed cards faceup. If that card is an event, trash it.",
+        "title": "Assimilator",
+        "type_code": "resource",
+        "uniqueness": false
+    },
+        {
```
Comment (gereons) [inline pack/ss.json]: Please align this to the line above

**Comment by Alsciende** [comment]
Deployed, thanks!

**Pull request merged**